.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--- Provide a general summary of your changes in the title above -->
+
+## Purpose
```

Comment:
some examples:

this PR:
purpose: add issue & PR templates to the repository
motivation and context: we get issues & PRs that lack information that would be useful to us as developers

for https://github.com/supercollider/supercollider/pull/3013:
purpose: This will insert keys instead of appending them.
motivation and context: no clue

for https://github.com/supercollider/supercollider/pull/2861:
purpose: (see title)
motivation and context: fix for #2441 and #2444.

does that help? do you want to merge both sections together? both pieces of information are helpful here. the 'purpose' prompt could note that this may just be the title of the PR. what do you think?